File: Mensuration/Square and Cube/Area/AoS.c
//Area of a Circle
#include<stdio.h>
#include<math.h>
void main(){
    int r;
    float area=0.0;
    printf("Enter the Radius : ");
    scanf("%d",&r);
    area = M_PI*pow(r,2);
    printf("Area : %0.4f",area);
}

File: Mensuration/Square and Cube/Area/AoS.java
//Area of a Square
import java.util.Scanner;
public class AoS{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter Side : ");
        int s=sc.nextInt();
        sc.close();
        int area=Math.pow(s,2);
        System.out.println("Area : "+area);
    }
}
